cocos/core/assets/sprite-frame.ts

```diff
@@ -479,11 +504,13 @@ export class SpriteFrame extends Asset {
 
     protected _capInsets = [0, 0, 0, 0];
 
-    protected _atlasUuid: string = '';
+    protected _atlasUuid = '';
     // @ts-expect-error
     protected _texture: TextureBase | RenderTexture;
 
-    protected _flipUv = false;
+    protected _flipUvY = false;
+
+    protected _flipUvX = false;
```

Comment:
Please make the name same as public property

Reply:
Okay